test/common.py

```diff
@@ -2694,7 +2705,7 @@ def btest(self, filename, expected=None,
             post_build=None,
             cflags=None,
             timeout=None,
-            extra_tries=0,
+            extra_tries=None,
```

Comment:
Or even simplest, maybe this could be
```py
extra_tries = EMTEST_RETRY_FLAKY if self.flaky else 0,
```
and the decorator would assign
```
self.flaky = True
```